packages/backend-function/src/factory.ts

```diff
@@ -306,6 +306,7 @@ class AmplifyFunction
         bundling: {
           format: OutputFormat.ESM,
           banner: bannerCode,
+          bundleAwsSDK: true,
```

Comment:
does this bundle the entire sdk or just the packages used in the lambda code?

Reply:
The way this works is:
1. Package must be installed locally, otherwise esbuild fails compilation
2. SDK packages are now not deny-listed
3. The mismatch between local and remote SDK disappears. (which looks like a bug from customer POV)

(this is also reasoning why we don't need to expose this as knob).